src/PoCs/PopoverThirtyMinutes.tsx

```diff
@@ -0,0 +1,99 @@
+import { Popover, ProgressStep, ProgressStepper, Button, Alert } from "@patternfly/react-core";
+import React, {
+  FunctionComponent,
+  useState,
+  VoidFunctionComponent,
+} from "react";
+
+type KafkaStatusPopoverProps = {
+    shouldStartOpen?: boolean;
+  } & KafkaStatusPopoverBodyProps;
+  
+  export const KafkaStatusPopover: FunctionComponent<KafkaStatusPopoverProps> = ({
+    shouldStartOpen = false,
+    currentState,
+    children,
+  }) => {
+    const [isVisible, setIsVisible] = useState(shouldStartOpen);
+  
+    return (
+      <Popover
+        headerContent="Creating instance"
+        bodyContent={<KafkaStatusPopoverBody currentState={currentState} />}
+        isVisible={isVisible}
+        shouldOpen={() => setIsVisible(true)}
+        shouldClose={() => setIsVisible(false)}
+      >
+        {children}
+      </Popover>
+    );
+  };
+  
+  type States = "pending" | "provisioning" | "preparing" | "ready" ;
+  
+  type KafkaStatusPopoverBodyProps = {
+    currentState: States;
+  };
+  export const KafkaStatusPopoverBody: VoidFunctionComponent<
+    KafkaStatusPopoverBodyProps
+  > = ({ currentState }) => {
+    const order: States[] = ["pending", "provisioning", "preparing", "ready"];
+    const currentStep = order.findIndex((v) => v === currentState);
+    console.log(currentStep);
+    return (
+      <div>
+        <Alert variant="danger" isInline isPlain title="This is taking longer than expected." />
+        <p>In the mean time you can create a Service account under the <a href="http://www.google.com/">Connections tab. </a>If your Kafka instance continues to take longer than expected, open a <a href="http://www.google.com/">support case.</a> </p>
```

Comment:
```suggestion
        <p>In the meantime, you can create a service account under the <a href="http://www.google.com/">Connections tab. </a>If your Kafka instance continues to take longer than expected, <a href="http://www.google.com/">open a support case.</a> </p>
```

Reply:
I moved the hyperlink slightly so that "open a support case" is all hyperlinked. That way, if a screenreader reads the link out of context, it will still make sense to the user.